hera_qm/utils.py

```diff
@@ -722,10 +724,13 @@ def apply_yaml_flags(uv, a_priori_flag_yaml, lat_lon_alt_degrees=None, telescope
             antnum = ant[0]
         if issubclass(uv.__class__, UVData):
             blt_selection = np.logical_or(uv.ant_1_array == antnum, uv.ant_2_array == antnum)
-            uv.flag_array[blt_selection, :, :, pol_selection] = True
+            if np.count_nonzero(blt_selection) > 0:
```

Comment:
I prefer `np.any(blt_selection)` but that's stylistic.

Reply:
I think your way is a bit more concise so I'm happy to change this.